ynput/OpenPype
# Resolve: load clip to timeline at set time

## Changelog Description
It is possible to load clip to correct place on timeline.

# Todo
- [x] add option to loader plugin to add sequencial loading as an opose to in place loading
- [ ] ~seek the way to add it always to new layer as it is done in Hiero. layer name inherited from product name~

## Testing notes:
1. Open testing timeline in project
2. load clip from loader and see it is added at correct place on timeline which had been captured during original editorial ingest. 
2. follow this step
